doc/filters/slug.rst

```diff
@@ -0,0 +1,29 @@
+``slug``
+========
+
+The ``slug`` filter transforms a given string into another string that
+only includes safe ASCII characters. It does so by using the method by
+the same name in Symfony's `AsciiSlugger <https://symfony.com/doc/current/components/string.html#slugger>`_.
+
+Here is an example:
+
+.. code-block:: twig
+
+    {{ 'Wôrķšƥáçè ~~sèťtïñğš~~'|slug }}
+    Workspace-settings
+
+The default separator between words is a dash (``-``), but you can 
+define a selector of your choice by passing it as an argument:
+
+.. code-block:: twig
+
+    {{ 'Wôrķšƥáçè ~~sèťtïñğš~~'|slug('/') }}
+    Workspace/settings
+
+The slugger automatically detects the language of the original
+string, but you can also specify it explicitly using the second
+argument:
+
+.. code-block:: twig
+
+    {{ '...'|slug('-', 'ko') }}
```

Comment:
Can you document the argument names (as done with other filters)? This helps use named arguments instead of positional ones.